pilz_psenscan_test_area/setup_workspace.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -x
+
+create_catkin_ws()
+{
+  mkdir -p "$SRC_DIR" 2>&1
+}
+
+clone_repo()
+{
+  ls "$REPO_DIR" > /dev/null 2>&1 && rm -rf "$REPO_DIR" 2>&1
+  git clone --branch "$TARGET_BRANCH" "https://github.com/PilzDE/$REPO_NAME.git" "$REPO_DIR" 2>&1 || exit 1
+}
+
+log_repo_info()
+{
+  cd "$REPO_DIR"
+  branch=$(git branch 2> /dev/null | sed -e '/^[^*]/d' -e 's/* \(.*\)/\1/')
+  commit_hash=$(git log -1 | sed -e '/^[^c]/d' -e 's/commit \(.*\)/\1/')
```

Comment:
A short comment to document what these lines do would be helpful.

Reply:
done